bundles/org.openhab.binding.miio/src/main/resources/OH-INF/thing/vacuumThing.xml

```diff
@@ -299,6 +315,18 @@
 		<item-type>String</item-type>
 		<label>Vacuum Room [room#]</label>
 	</channel-type>
+	<channel-type id="collectdust" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Start collect the dust at the station</label>
+	</channel-type>
+	<channel-type id="cleanmopstart" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Start mop wash</label>
```

Comment:
```suggestion
		<label>Start Mop Wash</label>
```

Reply:
Thanks I need to get used to the writings you are using here.. :-)